src/wazuh_modules/vulnerability_scanner/tests/unit/eventSendReport_test.cpp

```diff
@@ -57,7 +57,6 @@ const std::string SYNC_STATE_MSG {
                          "version": "0.1"
                     },
                     "index": "e051a99ad3950084b538c6538c0fcad0f1c4c713",
-                    "timestamp": ""
```

Comment:
Question: just to understand, why do we remove this?
It has to do with the schemas, but I don't see it.

Reply:
The timestamp was removed from fbs